Bottom: 00e9ae338eaa0bda25b4c61f9b39e5c577ba37ab
Top:    bde164abd5c5f7230d49ac10416f981fa184b548
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 17:13:01 +0000

Refresh of gtk-gl-play-with-layer-translu

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e4b55d4..0f47155 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -691,9 +691,11 @@ ghid_fill_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 void
 ghid_thindraw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 {
-  double old_alpha_mult = gc->alpha_mult;
+  gtkGC gtk_gc = (gtkGC)gc;
+
+  double old_alpha_mult = gtk_gc->alpha_mult;
   common_thindraw_pcb_polygon (gc, poly, clip_box);
-  ghid_set_alpha_mult (gc, gc->alpha_mult * 0.25);
+  ghid_set_alpha_mult (gc, gtk_gc->alpha_mult * 0.25);
   gui->graphics->fill_pcb_polygon (gc, poly, clip_box);
   ghid_set_alpha_mult (gc, old_alpha_mult);
 }
